docs/src/guides/tables.md

```diff
@@ -41,11 +41,10 @@ A Table is a collection of Records in a LanceDB Database. You can follow along o
     ```python
     import pandas as pd
 
-    data = pd.DataFrame({
-        "vector": [[1.1, 1.2, 1.3, 1.4], [0.2, 1.8, 0.4, 3.6]],
-        "lat": [45.5, 40.1],
-        "long": [-122.7, -74.1]
-    })
+    data = pd.DataFrame([
+        {"vector": [1.1, 1.2, 1.3, 1.4], "lat": 45.5, "long": -122.7},
+        {"vector": [0.2, 1.8, 0.4, 3.6], "lat": 40.1, "long": -74.1},
+    ])
```

Comment:
Let's leave this one as is. pandas dataframes make sense to be columnar.

Reply:
Reverted.